[DOCS] Replaces outdated integration guides with redirects to versioned documentation (#8173)
great-expectations/great_expectations: docs/docusaurus/docs/deployment_patterns/how_to_use_great_expectations_with_ydata_synthetic.md

sidebar_custom_props: { icon: 'img/integrations/ydata_synthetic_icon.png' }
---

> YData-Synthetic is an open-source synthetic data engine. Using different kinds of Generative Adversarial Networks (GANS), the engine learns patterns and statistical properties of original data. It can create endless samples of synthetic data that resemble the original data.

Reviewer: ```suggestion
YData-Synthetic is an open-source synthetic data engine. It uses Generative Adversarial Networks (GANS) to identify patterns and statistical properties in the originating data to help it create synthetic data samples that resemble the originating data.
```

:::info 

To use Great Expectations (GX) with YData-Synthetic, see [How to use GX with YData-Synthetic](/docs/0.15.50/deployment_patterns/how_to_use_great_expectations_with_ydata_synthetic) in version 0.15.50 of the documentation.

> For more information about implementing and using Ydata-Synthetic, see the [YData-Synthetic site](https://github.com/ydataai/ydata-synthetic).

Reviewer: ```suggestion
For more information about implementing and using Ydata-Synthetic, see the [YData-Synthetic site](https://github.com/ydataai/ydata-synthetic).
```
The previous instance was YData. Confirm the correct format and use it consistently.
